tools/salmon/macros.xml

```diff
@@ -8,6 +8,10 @@
             <requirement type="package" version="1.2">seqtk</requirement>
             <requirement type="package" version="0.14.1">salmon</requirement>
             <requirement type="package" version="1.9">samtools</requirement>
+            <requirement type="package" version="0.2.0">vpolo</requirement>
+            <requirement type="package" version="1.17.2">numpy</requirement>
```

Comment:
where is all of this used?

Reply:
It's strange, but for some reason, even though vpolo's bioconda wrapper works properly, if I don't also import numpy, etc. in the requirements, the tool throws a fit